Content.Client/_Starlight/Weapon/WeaponDismantleOnShootSystem.cs

```diff
@@ -0,0 +1,24 @@
+using Content.Shared.Starlight.Weapon.Systems;
+using System.Numerics;
+using Content.Shared._Starlight.Weapon.Components;
+using Content.Shared.Inventory;
+using Content.Shared.Popups;
+using Content.Shared.Tag;
+using Content.Shared.Throwing;
+using Content.Shared.Weapons.Ranged.Systems;
+using Robust.Shared.Physics.Systems;
+
+//linq
+using System.Linq;
+using Content.Shared.Weapons.Ranged.Events;
+using Robust.Shared.Random;
+using Content.Shared.Damage;
+using Content.Shared.Wieldable;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Player;
+using Robust.Shared.Audio;
+
+namespace Content.Client.Starlight.Weapon.Systems;
+public sealed partial class WeaponDismantleOnShootSystem : SharedWeaponDismantleOnShootSystem
+{
+}
```

Comment:
Remove all these unneeded namespaces. Fix Starlight namespacing.

```suggestion
using Content.Shared._Starlight.Weapon.Systems;

namespace Content.Client._Starlight.Weapon.Systems;

public sealed partial class WeaponDismantleOnShootSystem : SharedWeaponDismantleOnShootSystem;
```